docs/source/algorithms/StatisticsOfTableWorkspace-v1.rst

```diff
@@ -32,7 +32,7 @@ Usage
     for idx in range(stats.rowCount()):
         stat_name = stats.column('statistic')[idx]
         stat_value = stats.column('a')[idx]
-        print '%s of column \'a\' is %.3f' % (stat_name, stat_value)
+        print('%s of column \'a\' is %.3f' % (stat_name, stat_value))
```

Comment:
printing a dict can give different order in python 3
```
Expected:
    standard_deviation of column 'a' is 1.118
    minimum of column 'a' is 1.000
    median of column 'a' is 2.500
    maximum of column 'a' is 4.000
    mean of column 'a' is 2.500
Got:
    standard_deviation of column 'a' is 1.118
    maximum of column 'a' is 4.000
    minimum of column 'a' is 1.000
    mean of column 'a' is 2.500
    median of column 'a' is 2.500
```

Reply:
Interesting, this is a problem with the algorithm not the test.  Still I'll fix it anyway.